app/src/test/java/org/simple/clinic/registration/facility/RegistrationFacilitySelectionScreenControllerTest.kt

```diff
@@ -204,7 +229,11 @@ class RegistrationFacilitySelectionScreenControllerTest {
     setupController()
     uiEvents.onNext(RegistrationFacilityConfirmed(facility1.uuid))
 
+    verify(screen).showProgressIndicator()
```

Comment:
Using `inOrder` in the first place was a mistake since it makes tests covariant, i.e., dependent on implementation of the code that is being tested. Removed it to make migrating easier.

The right level of tests to use for a migration of this sort is UI tests, but since we don't have them and it would be a lot of effort to add them, we settle for using the current suite of tests as integration tests of a sort and pair it with manual testing of the screen as part of the migration.

Reply:
So if `inOrder` has this drawback, we should be removing inOrder(..) from other tests as well? Or this was intentional?

Reply:
Yes, we should be. I have done it in a few places whenever this was causing too much trouble during migration. Those which do not, I have left them as is.

Reply:
@msasikanth This conversation is relevant if you come across issues with `inOrder` while migrating to Mobius.